sdk/java-sdk-protobuf-testkit/src/main/scala/kalix/javasdk/testkit/impl/EventingTestKitImpl.scala

```diff
@@ -322,15 +335,23 @@ private[testkit] class TopicImpl(
   override def publish(message: ByteString, metadata: SdkMetadata): Unit =
     brokerProbe.emit(message, metadata)
 
-  override def publish[T <: GeneratedMessageV3](message: TestKitMessage[T]): Unit =
-    publish(message.getPayload.toByteString, message.getMetadata)
+  override def publish(message: TestKitMessage[?]): Unit = message.getPayload match {
```

Comment:
I don't remember to have ever seen `[?]` usage in Scala. Is that the same as `[_]`?

Reply:
Me neither, it was a typo but now I learnt that's what we'll actually need to use in scala 3... see [this](https://docs.scala-lang.org/scala3/reference/changed-features/wildcards.html#)